admin/data/sql/migrations/20190320145618.do.grant-census-import-user-permissions.js

```diff
@@ -3,12 +3,13 @@
 const config = require('../../../config')
 
 module.exports.generateSql = function () {
-  if (config.Sql.Azure.Scale) {
-    return `
+  return `
+    CREATE SCHEMA [mtc_census_import];
```

Comment:
@jon-shipley shouldn't you undo the schema creation on the undo statement?

Reply:
You were too fast!  Now added.